src/pages/PostsList/components/PostPreview.jsx

```diff
@@ -1,5 +1,38 @@
+import React from "react";
+import IconText from "./IconText";
+import { List, Typography, Button } from "antd";
+import { LikeOutlined } from "@ant-design/icons";
+
+const { Paragraph } = Typography;
+
 const PostPreview = ({ post }) => {
-  return <div>PostPreview</div>;
+  return (
```

Comment:
If you only return sth inside component, you can skip curly brackets and return statement.
```suggestion
  const PostPreview = ({ post }) => <List.Item ... >
```